src/plugins/emoteCloner/index.tsx

```diff
@@ -56,7 +56,7 @@ function getUrl(data: Data) {
     if (data.t === "Emoji")
         return `${location.protocol}//${window.GLOBAL_ENV.CDN_HOST}/emojis/${data.id}.${data.isAnimated ? "gif" : "png"}`;
 
-    return `${window.GLOBAL_ENV.MEDIA_PROXY_ENDPOINT}/stickers/${data.id}.${StickerExt[data.format_type]}`;
+    return `${window.GLOBAL_ENV.MEDIA_PROXY_ENDPOINT}/stickers/${data.id}.${StickerExt[data.format_type]}?size=2048`;
```

Comment:
I've been stumbling upon this issue too and I think it'd be nice to use the highest quality possible to preserve fidelity, and since this is the highest option Discord provides afaik

```suggestion
    return `${window.GLOBAL_ENV.MEDIA_PROXY_ENDPOINT}/stickers/${data.id}.${StickerExt[data.format_type]}?size=4096`;
```

Reply:
the higher the better, just pushed that change rn

Reply:
don't stickers cap out at 320px anyway?

Reply:
> don't stickers cap out at 320px anyway?

Not sure, but I don't think it does